parlai/core/dialog_teacher.py

```diff
@@ -153,12 +153,14 @@ class DialogData(object):
     (x, ...), new_episode?
 
     Where...
-    x is a query and possibly context
+    - x is a query and possibly context
     ... can contain additional fields, specifically
-        y is an iterable of label(s) for that query
-        r is the str reward for getting that query correct
-        c is an iterable of label candidates that the student can choose from
-    new_episode? is a boolean value specifying whether that example is the start
+      - y is an iterable of label(s) for that query
+      - r is the str reward for getting that query correct
+      - c is an iterable of label candidates that the student can choose from
+      - i is a str path to an image on disk, which will be loaded by the data
```

Comment:
yeah need to confirm exactly how we want to do this, but the basic solution I set up in load_data was that a folder should exist in the same directory as the images which contains the name of the preprocessor ('vgg') which contains a preprocessed version of the images

see load_data below for that